lib/InternalContactsFieldArray/UserField.js

```diff
@@ -125,7 +133,7 @@ export default class UserField extends React.Component {
               </span>
             </KeyValue>
           </Col>
-          <Col md={4} xs={6}>
+          <Col md={6} xs={6}>
```

Comment:
I think you can avoid passing both xs and md when they are the same value. Please double check